process/block/preprocess/transactions.go

```diff
@@ -424,8 +433,27 @@ func (txs *transactions) CreateAndProcessMiniBlock(sndShardId, dstShardId uint32
 	strCache := process.ShardCacherIdentifier(sndShardId, dstShardId)
 	txStore := txs.txPool.ShardDataStore(strCache)
 
+	var (
```

Comment:
for some reason I don't like this construction inside methods, can you add a 'var' for each of the variables?

Reply:
Done